consensus/state.go

```diff
@@ -1975,7 +1975,7 @@ func (cs *State) tryAddVote(vote *types.Vote, peerID p2p.NodeID) (bool, error) {
 			// 2) not a bad peer? this can also err sometimes with "Unexpected step" OR
 			// 3) tmkms use with multiple validators connecting to a single tmkms instance
 			// 		(https://github.com/tendermint/tendermint/issues/3839).
-			cs.Logger.Info("failed attempting to add vote", "err", err)
+			cs.Logger.Debug("failed attempting to add vote", "err", err)
```

Comment:
A user requested this be INFO.

Reply:
oh, weird. will change it back. I saw it polluting the logs on cosmoshub mainnet so decided to move it to debug.